scrape/apps.go

```diff
@@ -110,11 +110,14 @@ type OAuthApp struct {
 
 // AppManifest represents a GitHub App manifest, used for preconfiguring
 // GitHub App configuration.
+// c.f. https://docs.github.com/en/apps/sharing-github-apps/registering-a-github-app-from-a-manifest
```

Comment:
Thank you for adding the link, although now I'm slightly confused.
According to this link, the field should be:

```go
CallbackURLs []string `json:"callback_urls,omitempty"`
```

Could you please try this out in your local code and compare it to the one you added and see if there is a difference and/or if both of them work? (In which case, should we actually _add_ both?!?!?)

Reply:
Both works. I haven't tested the behavior when using both.

Since `CallbackURLs` is a superset of `CallbackURL`, implementing only the former seems best to avoid user confusion.